packages/web/src/services/http/http.service.ts

```diff
@@ -20,8 +20,10 @@ http.interceptors.response.use(
   response => response,
   error => {
     if (error.response.status === 401) {
-      localStorage.clear();
-      window.location.href = '/signin';
+      if (window.location.pathname !== '/signin') {
```

Comment:
What about to add preventInterceptor boolean flag to the config of signing request?
if (error.response.status === 401 && !error.response.config.preventUnauthorizedInterceptor)

Reply:
Yeah, this would be smart, but it is not easily achievable in a type safe way before this one (release/1.0.0-beta.1):
https://github.com/axios/axios/pull/1964/files/678feff772741704d3c2ef2d127055ba59df3382

Reply:
Or maybe it is...

Reply:
Yep, [that's better](https://github.com/dubbha/croton/pull/103/commits/30c7eb575af67b185dd55d3301e16145f8d5efd9), thanks!